docker/circleci/Dockerfile

```diff
@@ -1,5 +1,5 @@
 #https://github.com/CircleCI-Public/circleci-dockerfiles/blob/master/golang/images/1.12.4/Dockerfile
-FROM golang:1.12.4
+FROM golang:1.13
```

Comment:
Just curious, but does it change anything for us ?

Reply:
Yes, the modules dependencies imports for operator-sdk :  https://github.com/erdrix/casskop/blob/upgrade_operator-sdk/docker/circleci/Dockerfile#L87 failed during the image build :

```logs
Makefile:67: recipe for target 'tidy' failed
make: *** [tidy] Error 1
```